app/views/admin/configs/show.html.erb

```diff
@@ -130,9 +130,40 @@
                              class: "form-control selectpicker" %>
               <div class="alert alert-info"><%= Constants::SiteConfig::DETAILS[:authentication_providers][:description] %></div>
             </div>
+            <div class="form-group">
+              <%= admin_config_label :twitter_key %>
+              <%= f.text_field :twitter_key,
+                               class: "form-control",
+                               value: SiteConfig.twitter_key,
+                               placeholder: Constants::SiteConfig::DETAILS[:twitter_key][:placeholder] %>
```

Comment:
As is, these values are still going to require an app restart to take affect. Should we mention something about that here so people dont assume as soon as they are changed they are being used?

Reply:
Ah... I didn't realize it still worked this way. I was thinking that the instructions were passed in to execute at runtime, even if it was instructed at boot.

I added a notice about this for now, but I still think we can make further adjustments to make sure this just works. Need to read through the omniauth instructions more clearly 😄